src/messages/trial.messages.js

```diff
@@ -165,7 +165,7 @@ const trialMessages = defineMessages({
   },
   startInstallTrial: {
     id: 'trial.startInstallTrial',
-    defaultMessage: 'You may need to wait up to 30 minutes to access <a>automation hub</a> services after the trial subscription is activated.'
+    defaultMessage: 'You may need to wait up to 30 minutes to access <a href="https://cloud.redhat.com/ansible/automation-hub">automation hub</a> services after the trial subscription is activated.'
```

Comment:
cloud URLs are automatically redirected to console ones. Let's use that here directly
```suggestion
    defaultMessage: 'You may need to wait up to 30 minutes to access <a href="https://console.redhat.com/ansible/automation-hub">automation hub</a> services after the trial subscription is activated.'
```

Reply:
Not all clouds urls go to console. Only the ones specified when we made the cutover. For example, this one shouldn't be redirecting.

Reply:
I reverted the changes I made